chain/vm/mkactor.go

```diff
@@ -30,6 +30,10 @@ var EmptyObjectCid cid.Cid
 
 // TryCreateAccountActor creates account actors from only BLS/SECP256K1 addresses.
 func TryCreateAccountActor(rt *Runtime, addr address.Address) (*types.Actor, aerrors.ActorError) {
+	if err := rt.chargeGasSafe(PricelistByEpoch(rt.height).OnCreateActor()); err != nil {
```

Comment:
This reorg is safe because the only way we would end up a different amount of gas is if something failed, and then we would end up charging all the gas.